screenshot-iframe.js

```diff
@@ -0,0 +1,34 @@
+import {storageHost} from './constants';
+
+export const getScreenshotBlob = async (reqUrl, hash, ext, type, width, height) => {
```

Comment:
Change name of function to generatePreview

Reply:
Done